core/src/it/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -146,6 +146,20 @@ class BasicWorkflow extends EnsimeSpec
             case _ => false
           }
 
+          //-----------------------------------------------------------------------------------------------
+          // uses of symbol at point with a symbol defined in classfiles
+
+          project ! TypecheckFilesReq(List(Left(fooFile)))
+          expectMsg(VoidResponse)
+
+          project ! UsesOfSymbolAtPointReq(Left(fooFile), 162) // point on String.length
+          expectMsgType[ERangePositions].positions should contain theSameElementsAs List()
+
+          asyncHelper.fishForMessage() {
```

Comment:
we've found these `fishForMessage` waiting blocks to be very unreliable. Please try to work out exactly what messages are being returned and wait for them. It is probably safer to put your test at the very end.